pkg/machine/machine_common.go

```diff
@@ -54,3 +54,71 @@ func AddSSHConnectionsToPodmanSocket(uid, port int, identityPath, name, remoteUs
 	}
 	return nil
 }
+
+// WaitAPIAndPrintInfo prints info about the machine and does a ping test on the
+// API socket
+func WaitAPIAndPrintInfo(forwardState APIForwardingState, name, helper, forwardSock string, noInfo, isIncompatible, rootful bool) {
+	suffix := ""
+	if name != DefaultMachineName {
+		suffix = " " + name
+	}
+
+	if isIncompatible {
+		fmt.Fprintf(os.Stderr, "\n!!! ACTION REQUIRED: INCOMPATIBLE MACHINE !!!\n")
```

Comment:
Nit and not something to tackle in this PR.

golang supports raw strings where we can specify new lines which would be _much_ easier to read and maintian than printing each line separately as done below.